test/bdd/pkg/internal/httputil/httputil.go

```diff
@@ -181,10 +174,3 @@ func WithSigner(signer requestSigner) Opt {
 		o.signer = signer
 	}
 }
-
-// WithParsedResponse specifies type to unmarshal response body.
-func WithParsedResponse(r interface{}) Opt {
```

Comment:
consider cleaning up `parsedResponse`-related behavior as well (the same considerations as for `bearerToken` case above)

Reply:
removed